chart-value-examples/azure/values.yaml

```diff
@@ -0,0 +1,25 @@
+image:
+  repository: hub.docker.nexus3.linkurious.net/linkurious/linkurious-enterprise
```

Comment:
@enys are you sure you want to keep the URL here?

Reply:
We do not. Good catch

Reply:
This has been fixed, thx